enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/SaveButton.tsx

```diff
@@ -15,9 +15,9 @@ export const SaveButton = ({
     ? question && question.isQueryDirtyComparedTo(originalQuestion)
     : true;
 
-  return (
-    <Button disabled={!isQuestionChanged || !canSave} onClick={onClick}>
-      Save
-    </Button>
-  );
+  if (!isQuestionChanged || !canSave) {
+    return null;
+  }
+
+  return <Button onClick={onClick}>Save</Button>;
```

Comment:
This is based on [Maz's feedback in this private Loom](https://www.loom.com/share/aafc92798b814f56bd989765153920dc):

> If you're looking at a saved, unmodified question, I don't think the Save button should be visible at all, until/unless the query or visualization is modified. This applies to saving questions in general. I can't really think of a situation where we'd want to show the Save button but not have it be enabled.